apple/Sources/FerrostarSwiftUI/Views/Maneuver/ManeuverInstructionView.swift

```diff
@@ -1,10 +1,13 @@
+import CoreLocation
 import FerrostarCoreFFI
+import MapKit
 import SwiftUI
 
 /// A Customizable Maneuver Instruction View.
 public struct ManeuverInstructionView<ManeuverView: View>: View {
     private let text: String
-    private let distanceRemaining: String?
+    private let distanceToNextManeuver: CLLocationDistance?
+    private let distanceFormatter = MKDistanceFormatter()
```

Comment:
Minor: Not sure if it's an issue, but this requires the view to have some "smarts" and that feels a tiny bit wrong. Feels like we should be able to have an `MKDistanceFormatter` at a higher level that gets configured once rather than creating one every time we re-compute a `ManeuverInstructionView`.